src/client/src/pages/settings/admin/workgroupDetail.tsx

```diff
@@ -29,11 +31,34 @@ export const WorkgroupDetail: FC = () => {
   const { data: selectedWorkgroup } = useSelectedWorkgroup(parseInt(id));
   const {
     removeAllRoles: { mutate: removeAllWorkgroupRolesForUser },
+    update: { mutate: updateWorkgroup },
   } = useWorkgroupMutations();
   const { showPrompt } = useContext(PromptContext);
   const [filterModel, setFilterModel] = useState<GridFilterModel>();
   const handleFilterModelChange = useCallback((newModel: GridFilterModel) => setFilterModel(newModel), []);
 
+  const formMethods = useForm({
+    defaultValues: selectedWorkgroup,
+  });
+
+  const changeName = useCallback(
```

Comment:
Q: Wäre es allenfalls sinnvoll die das debounce im useMutation hook zu machen und die Daten in der query optimistically upzudaten? (In useUserMutations haben wir bereits eine Mutation mit optimistic update und rollback on error. )
So müssen sich die einzelnen Komponenten die die Mutations nutzen wollen nicht ums debouncen kümmern?